hr_timesheet_sheet/tests/test_hr_timesheet_reviewer_no_email.py

```diff
@@ -0,0 +1,107 @@
+from .test_hr_timesheet_sheet import TestHrTimesheetSheet
+
+
+class TestHrTimesheetReviewerNoEmail(TestHrTimesheetSheet):
+    @classmethod
+    def setUpClass(cls):
+        super(TestHrTimesheetReviewerNoEmail, cls).setUpClass()
+        cls.timesheet_model_obj = cls.env["hr_timesheet.sheet"]
+        cls.mail_message_obj = cls.env["mail.message"]
+        cls.message_dict = {
+            "subject": "Test Message #1",
+            "body": "Test Message Content",
+            "model": "hr_timesheet.sheet",
+            "message_type": "notification",
+        }
+
+    def test_notify_compute_recipients(self):
+        """Tests notify compute recipients
+
+        it's should prevent sending emails to the reviewer
+        """
+        timesheet = self.timesheet_model_obj.with_user(self.user_4).create(
+            {
+                "name": "Test Timesheet",
+                "employee_id": self.department_manager.id,  # user_3
+            }
+        )
+        msg_dict = self.message_dict.copy()
+        # Message created when creating timesheet
+        message_create = self.mail_message_obj.search(
+            [("model", "=", "hr_timesheet.sheet"), ("res_id", "=", timesheet.id)]
+        )
+        # send timesheet to review
+        timesheet.with_user(self.user_3).action_timesheet_confirm()
+        self.assertEqual(timesheet.state, "confirm", msg="state must be changed")
```

Comment:
Better will change assert message on `Timesheet state must be confirm`

Reply:
resolved